### PR TITLE
Add DW ray image to RHOAI additional images

### PR DESCRIPTION
**Pull request by oksanabaza**
Added the current SHA for the quay.io/modh/ray:2.35.0-py39-cu121 image into RHOAI 2.14 additional images bundle

### CODE REVIEW

**Review by astefanutti — COMMENTED** [review]
That looks good, but it seems there is already a new SHA published. I'd like to clarify that point first and understand if we can go with this one or should use the latest.

**Comment by Fiona-Waters** [comment]
@oksanabaza could you update to `quay.io/modh/ray@sha256:83084b89885232e5733027b2ee45b83d3642ce32ae0b8e18b79ed45b6e734283` please

File: rhoai-disconnected-images.yaml [hunk 1]
```diff
@@ -1,2 +1,3 @@
 additional-images:
 - quay.io/modh/fms-hf-tuning@sha256:8edea6f0f9c4c631cdca1e1c10abf0d4b994738fde78c40d48eda216fdd382f5
+- quay.io/modh/ray@sha256:83084b89885232e5733027b2ee45b83d3642ce32ae0b8e18b79ed45b6e734283
```
Comment (astefanutti) [inline rhoai-disconnected-images.yaml]: I think that `sha256:8b9e0efa3ae0e1862d66c02ffa82b80830334a5f8af12deb96d4f2f8babce5fe` may be the correct one for CUDA. @Fiona-Waters

Comment (Fiona-Waters) [reply rhoai-disconnected-images.yaml]: Yes it should be `quay.io/modh/ray@sha256:8b9e0efa3ae0e1862d66c02ffa82b80830334a5f8af12deb96d4f2f8babce5fe`
Apologies for the confusion.

Comment (oksanabaza) [reply rhoai-disconnected-images.yaml]: thank you, I updated the image

**Review by astefanutti — APPROVED** [review]

**Pull request merged**